src/core/dom/index.js

```diff
@@ -536,3 +537,34 @@ export function dispatchCustomEvent(node, name, opt_data, opt_options) {
 export function containsNotSelf(parent, child) {
   return child !== parent && parent.contains(child);
 }
+
+/**
+ * Configures the supplied element to have a "fill content" layout. The
+ * exact interpretation of "fill content" depends on the element's layout.
+ *
+ * If `opt_replacedContent` is specified, it indicates whether the "replaced
+ * content" styling should be applied. Replaced content is not allowed to
+ * have its own paddings or border.
+ *
+ * @param {!Element} element
+ * @param {boolean=} opt_replacedContent
+ * @public @final
+ */
+export function applyFillContent(element, opt_replacedContent) {
+  element.classList.add('i-amphtml-fill-content');
```

Comment:
Is this a new function? I notice it's replaced in the CustomElement method, wasn't clear on if it was shared somewhere

Reply:
The function used to live in `CustomElement`, but I've moved it to core.

Reply:
Do we expect it's going to be used elsewhere though?

Reply:
Resolving this thread to consolidate related threads

Reply:
Yep! By the compiler